### PR TITLE
 Update and rename jobs.ch.xml to JobCloud.ch.xml

### PR DESCRIPTION
**Pull request by cschanaj**
remove the 'evil regex' in the code base...

### CODE REVIEW

File: src/chrome/content/rules/jobcloud.ch.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<ruleset name="jobcloud.ch">
+	<target host="jobcloud.ch" />
+	<target host="www.jobcloud.ch" />
+	<target host="cdn.jobcloud.ch" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/jobcloud.ch.xml]: Different content HTTP/HTTPS

File: src/chrome/content/rules/jobcloud.ch.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	<target host="confluence.jobcloud.ch" />
+	<target host="test.confluence.jobcloud.ch" />
+	<target host="img.jobcloud.ch" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/jobcloud.ch.xml]: Do you have a test URL for this? It's okay if you don't but it's better if you have one.

File: src/chrome/content/rules/jobcloud.ch.xml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<ruleset name="jobcloud.ch">
```
Comment (Bisaloo) [inline src/chrome/content/rules/jobcloud.ch.xml]: Please capitalize this as it i shown on the page title, ie `JobCloud.ch`

**Comment by Bisaloo** [comment]
Please also rename the file to `JobCloud.ch.xml` to match the new ruleset name.

**Comment by cschanaj** [comment]
@Bisaloo I can't find any test url for `img.jobcloud.ch`, it seems that they are not using it to serve the images. Renamed the rulesets and commented `cdn`. thanks!!

**Pull request merged**